libraries/AP_Gripper/AP_Gripper_Servo.cpp

```diff
@@ -10,19 +10,15 @@ extern const AP_HAL::HAL& hal;
 
 void AP_Gripper_Servo::init_gripper()
 {
-    // move the servo to the release position
-    release();
+    // move the servo to the neutral position
+    SRV_Channels::set_output_pwm(SRV_Channel::k_gripper, config.neutral_pwm);
 }
 
 void AP_Gripper_Servo::grab()
```

Comment:
I would have expected to see:
`config.state = AP_Gripper::STATE_GRABBING;`
In here somewhere.

Reply:
Thanks Peter, that is a stupid mistake. Thanks for pointing it out!